### PR TITLE
[Post] ES6 Object.assign

### PR DESCRIPTION
**Pull request by naholyr**

### CODE REVIEW

**Comment by MoOx** [comment]
@naholyr is on fire.
LGTM.
Thanks for this. Btw, you should make a PR to add an author json file (mainly for your bio). See https://github.com/putaindecode/putaindecode.io/tree/master/content/authors

File: content/fr/articles/js/es2015/object-assign.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+```
+
+## Cas particuliers
+
+### Erreurs lors de la copie
+
+Si une propriété de la source, ``Object.assign`` se comportera toujours comme en
+mode *strict* (par opposition au mode *WTF*), et lèvera l'erreur avant
+d'interrompre la copie.
+
```
Comment (Macxim) [inline content/fr/articles/js/es2015/object-assign.md]: Il ne manquerait pas quelque chose dans cette phrase ?

Comment (naholyr) [inline content/fr/articles/js/es2015/object-assign.md]: "est en lecture seule", je corrige ce soir

**Comment by MoOx** [comment]
@naholyr is this on purpose that you use `let` where you could use `const`?

**Comment by naholyr** [comment]
Yeah, highly arguable because I know there is no "technical" reason, but I use ``let`` to denote the fact that object's value will be modified, even if it stays the same reference I like **not** to use ``const`` for mutable data.

No justification at all for ``o1`` and ``o2`` in first example ;) Feel free to change it all the way (no time today), as you can see it's very personal :)

**Comment by Macxim** [comment]
Une raison de ne pas avoir intégré mes changements ? @naholyr https://github.com/naholyr/putaindecode.io/pull/1

**Comment by MoOx** [comment]
Feel free to push some update in master @Macxim.

**Comment by Macxim** [comment]
Done, thanks :+1:

**Pull request merged**